<?xml version="1.0"?>
<robot name="PlaDyPos" xmlns:xacro="http://www.ros.org/wiki/xacro">
	<xacro:property name="pi" value="3.1415926535897931" />
	<link name="base_link">
		<visual>
			<origin rpy="${pi} 0 0" xyz="0 0 0" />
			<geometry>
				<mesh filename="${mesh_path}" />
			</geometry>
		</visual>
		<!-- <collision> <origin rpy="0 0 0" xyz="0 0 0"/> <geometry> <mesh filename="${mesh_collision_path}"/> 
			</geometry> </collision> -->
		<inertial>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<mass value="25" />
			<inertia ixx="100.0" ixy="0.0" ixz="0.0" iyy="100.0" iyz="0.0"
				izz="100.0" />
		</inertial>
	</link>
	<gazebo>
		<static>true</static>
	</gazebo>
</robot>
